### PR TITLE
Stop redeclaring variables from params

### PR DESCRIPTION
**Pull request by mhaskel**
It really seems unnecessary.

### CODE REVIEW

**Comment by daenney** [comment]
E_NOMERGE says Github :(.

**Comment by mhaskel** [comment]
too many PRs touching too many things. try this.

File: manifests/pin.pp [hunk 1]
```diff
@@ -65,8 +63,8 @@
   $file_name = regsubst($title, '[^0-9a-z\-_\.]', '_', 'IG')
 
   $path = $order ? {
-    ''      => "${preferences_d}/${file_name}.pref",
-    default => "${preferences_d}/${order}-${file_name}.pref",
+    ''      => "${apt::params::preferences_d}/${file_name}.pref",
```
Comment (daenney) [inline manifests/pin.pp]: Couldn't his just be `apt::preferences_d`? We require `apt` and it inherits from `params`.t

File: manifests/pin.pp [hunk 2]
```diff
@@ -65,8 +63,8 @@
...
+    default => "${apt::params::preferences_d}/${order}-${file_name}.pref",
```
Comment (daenney) [inline manifests/pin.pp]: Idem

File: manifests/ppa.pp [hunk 3]
```diff
@@ -46,7 +45,7 @@
     exec { "add-apt-repository-${name}":
         environment => $proxy_env,
         command     => "/usr/bin/add-apt-repository ${options} ${name}",
-        unless      => "/usr/bin/test -s ${sources_list_d}/${sources_list_d_filename}",
+        unless      => "/usr/bin/test -s ${apt::params::sources_list_d}/${sources_list_d_filename}",
```
Comment (daenney) [inline manifests/ppa.pp]: Idem

File: manifests/ppa.pp [hunk 4]
```diff
@@ -56,14 +55,14 @@
         ],
     }
 
-    file { "${sources_list_d}/${sources_list_d_filename}":
+    file { "${apt::params::sources_list_d}/${sources_list_d_filename}":
```
Comment (daenney) [inline manifests/ppa.pp]: Idem

File: manifests/ppa.pp [hunk 5]
```diff
@@ -56,14 +55,14 @@
...
         ensure  => file,
         require => Exec["add-apt-repository-${name}"],
     }
   }
   else {
 
-    file { "${sources_list_d}/${sources_list_d_filename}":
+    file { "${apt::params::sources_list_d}/${sources_list_d_filename}":
```
Comment (daenney) [inline manifests/ppa.pp]: Idem

File: manifests/source.pp [hunk 6]
```diff
@@ -35,7 +32,7 @@
 
   file { "${name}.list":
     ensure  => $ensure,
-    path    => "${sources_list_d}/${name}.list",
+    path    => "${apt::params::sources_list_d}/${name}.list",
```
Comment (daenney) [inline manifests/source.pp]: Idem

**Comment by mhaskel** [comment]
done.

**Pull request merged**